extensions/modules/sql/src/main/java/org/exist/xquery/modules/sql/GetConnectionFunction.java

```diff
@@ -132,10 +127,25 @@ public Sequence eval(final Sequence[] args, final Sequence contextSequence) thro
         return new IntegerValue(SQLModule.storeConnection(context, connection), Type.LONG);
     }
 
+    /**
+     * This accesses the JDBC driver that has been loaded as an XAR package.
+     *
+     * @param jdbcDriver The classpath of the JDBC driver
+     * @return true if the driver is successfully loaded.
+     */
+    private boolean testClass(final StringValue jdbcDriver) {
+        try {
+            return (Class.forName(jdbcDriver.getStringValue()) != null);
```

Comment:
As far as I understand it @lcahlander since JDBC 4.0 using `Class.forName` is no longer required.

Reply:
no, unloading a class is  not possible (yet) ..... I typically have to restart the java process.